README.md

```diff
@@ -57,18 +57,7 @@ The default configuration uses both OpenAPI 3.0 rules as well as Spectral rules.
 
 ### Customization
 
-You can modify the behavior of the validator for your project to meet your preferred standards. Specific validation "rules" can be turned off, or configured to trigger an error, warning, info, or hint message in the validator output.
```

Comment:
I must have missed this section when I refactored the docs :)

Reply:
I also didn't even think about it!